test/libyul/yulOptimizerTests/fullSuite/abi2.yul

```diff
@@ -1108,11 +1108,11 @@
 //         {
 //             revert(value4, value4)
 //         }
-//         if gt(add(add(_1092, abi_decode_length_15_689), 0x20), dataEnd_56)
+//         if gt(add(add(_1092, abi_decode_length_15_689), 32), dataEnd_56)
```

Comment:
Can you explain this change?

Reply:
Ah ok, I think I get it: In an intermediate step, this was replaced by a variable that had the value `32` and then this replacement was undone in a later step.

Reply:
Yes - the CSE uses ``SyntacticallyEqual`` on expressions - and that used to compare the string representation of literals and now compares values for number literals, so this plus some hidden immediate steps should be the reason.